### PR TITLE
Add methods to get a system temp dir, create temp file and folder

### LINKED ISSUE #76: Methods to get a system temp directory, create temp file, create temp folder
I suggest that we create the following methods:

```csharp
static class Temporary
{
  public static AbsolutePath SystemTempDirectory();
  public static AbsolutePath CreateTempFile();
  public static AbsolutePath CreateTempFolder();
}
```

This resolves several problems of the existing `Path` class in the BCL:
- in `Path`, it's too easy to mix `GetTempFileName()` and `GetTempPath()`, and these have very different semantics;
- there's no standard function to create a temp folder;
- well, obviously, the standard function doesn't operate on `TruePath` types.

### PR DESCRIPTION
**Pull request by Illusion4**
Closes #76

### CODE REVIEW

File: TruePath/Temporary.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    }
+
+    /// <summary>
+    /// Creates a temporary folder with the specified prefix.
+    /// </summary>
+    /// <param name="prefix">An optional string to add to the beginning of the subdirectory name.</param>
+    /// <returns>An AbsolutePath representing newly created temporary folder</returns>
+    public static AbsolutePath CreateTempFolder(string? prefix = null)
+    {
+        var tempDirectoryInfo = Directory.CreateTempSubdirectory(prefix);
```
Comment (ForNeVeR) [inline TruePath/Temporary.cs:40]: Theoretically, any of these calls might return a relative path, so I'd suggest using `AbsolutePath.CurrentWorkingDirectory / tempPath` in all three cases.

**Review by ForNeVeR — CHANGES_REQUESTED** [review]
Other than a minor issue, good to go!

File: TruePath/Temporary.cs (hunk 1 above)
Comment (Illusion4) [reply TruePath/Temporary.cs]: Modified

**Review by ForNeVeR — APPROVED** [review]
Thanks!

**Pull request merged**